specs/phase1/beacon-chain.md

```diff
@@ -530,33 +533,75 @@ def get_indexed_attestation(beacon_state: BeaconState, attestation: Attestation)
     )
 ```
 
+#### `get_committee_count_delta`
+
+```python
+def get_committee_count_delta(state: BeaconState, start_slot: Slot, stop_slot: Slot) -> uint64:
+    """
+    Return the sum of committee counts between ``[start_slot, stop_slot)``.
+    """
+    committee_sum = 0
+    for slot in range(start_slot, stop_slot):
+        count = get_committee_count_at_slot(state, Slot(slot))
+        committee_sum += count
+    return committee_sum
+```
+
 #### `get_start_shard`
 
 ```python
 def get_start_shard(state: BeaconState, slot: Slot) -> Shard:
-    # TODO: implement start shard logic
-    return Shard(0)
+    """
+    Return the start shard at ``slot``.
+    """
+    current_epoch_start_slot = compute_start_slot_at_epoch(get_current_epoch(state))
+    active_shard_count = get_active_shard_count(state)
+    if current_epoch_start_slot == slot:
+        return state.current_epoch_start_shard
+    elif current_epoch_start_slot > slot:
```

Comment:
Should this be `slot > current_epoch_start_slot` ?  

`stop_slot` should be greater than `start_slot` for `get_committee_count_delta`